addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/TabletopKeypadHandler.java

```diff
@@ -0,0 +1,171 @@
+        LED12(92, "led12", "LED 12"),
+        LED13(93, "led13", "LED 13"),
+        LED14(94, "led14", "LED 14"),
+        LED15(95, "led15", "LED 15");
+
+        private final int id;
+        private final String channel;
+        private final String description;
+
+        COMPONENT(final int i, final String c, final String d) {
+            id = i;
+            channel = c;
+            description = d;
+        }
+
+        @Override
+        public int id() {
+            return this.id;
+        }
+
+        @Override
+        public String channel() {
+            return this.channel;
+        }
+
+        @Override
+        public String description() {
+            return this.description;
+        }
+
+    }
+
+    private static final List<COMPONENT> buttonGroup1 = Arrays.asList(COMPONENT.BUTTON1, COMPONENT.BUTTON2,
+            COMPONENT.BUTTON3, COMPONENT.BUTTON4, COMPONENT.BUTTON5);
+    private static final List<COMPONENT> buttonGroup2 = Arrays.asList(COMPONENT.BUTTON6, COMPONENT.BUTTON7,
+            COMPONENT.BUTTON8, COMPONENT.BUTTON9, COMPONENT.BUTTON10);
+    private static final List<COMPONENT> buttonGroup3 = Arrays.asList(COMPONENT.BUTTON11, COMPONENT.BUTTON12,
+            COMPONENT.BUTTON13, COMPONENT.BUTTON14, COMPONENT.BUTTON15);
+
+    private static final List<COMPONENT> buttonsBottomRL = Arrays.asList(COMPONENT.BUTTON16, COMPONENT.BUTTON17,
+            COMPONENT.LOWER3, COMPONENT.RAISE3);
+    private static final List<COMPONENT> buttonsBottomCRL = Arrays.asList(COMPONENT.LOWER1, COMPONENT.RAISE1,
+            COMPONENT.LOWER2, COMPONENT.RAISE2, COMPONENT.LOWER3, COMPONENT.RAISE3);
+
+    private static final List<COMPONENT> ledGroup1 = Arrays.asList(COMPONENT.LED1, COMPONENT.LED2, COMPONENT.LED3,
+            COMPONENT.LED4, COMPONENT.LED5);
+    private static final List<COMPONENT> ledGroup2 = Arrays.asList(COMPONENT.LED6, COMPONENT.LED7, COMPONENT.LED8,
+            COMPONENT.LED9, COMPONENT.LED10);
+    private static final List<COMPONENT> ledGroup3 = Arrays.asList(COMPONENT.LED11, COMPONENT.LED12, COMPONENT.LED13,
+            COMPONENT.LED14, COMPONENT.LED15);
+
+    private Logger logger = LoggerFactory.getLogger(TabletopKeypadHandler.class);
+
+    @Override
+    protected boolean isLed(int id) {
+        return (id >= 81 && id <= 95);
+    }
+
+    @Override
+    protected boolean isButton(int id) {
+        return (id >= 1 && id <= 25);
+    }
+
+    @Override
+    protected boolean isCCI(int id) {
+        return false;
+    }
+
+    @Override
+    protected void configureComponents(String model) {
+        String mod = model == null ? "null" : model;
+        this.logger.debug("Configuring components for keypad model {}", model);
+
+        switch (mod) {
+            default:
+                this.logger.warn("No valid keypad model defined ({}). Assuming model T15RL.", mod);
+            case "T15RL":
+                buttonList.addAll(buttonGroup3);
+                ledList.addAll(ledGroup3);
```

Comment:
I would add a `// fall through` comment to make it clear that these cases fall through on purpose.

Reply:
Agreed.  I added those comments.